src/libstore/parsed-derivations.cc

```diff
@@ -134,14 +134,8 @@ std::optional<nlohmann::json> ParsedDerivation::prepareStructuredAttrs(std::opti
     /* Add an "outputs" object containing the output paths. */
     nlohmann::json outputs;
     for (auto & i : drv.outputs) {
-        if (inputRewrites) {
-            /* The placeholder must have a rewrite, so we use it to cover both the
-               cases where we know or don't know the output path ahead of time. */
-            outputs[i.first] = rewriteStrings(hashPlaceholder(i.first), inputRewrites.value());
-        } else {
-            /* This case is only relevant for the nix-shell */
-            outputs[i.first] = hashPlaceholder(i.first);
-        }
+        /* This case is only relevant for the nix-shell */
```

Comment:
I suppose this comment doesn’t make sense anymore, dose it?

Reply:
Good catch, fixed!